web/src/app/loading/components/Spinner.js

```diff
@@ -59,7 +59,7 @@ export default class Spinner extends React.PureComponent {
 
     return (
       <span style={style}>
-        <CircularProgress size={this.props.text ? '1em' : null} />
+        <CircularProgress size={this.props.text ? '1em' : '5em'} />
```

Comment:
This seems quite large, can we use the default MUI size of `40px`, or `2.5em`? It's what the default should have been set to in Firefox before.